_pkgdown.yml

```diff
@@ -1,39 +1,39 @@
+authors:
```

Comment:
I don't think there is a reason to move the authors section within the file

Reply:
Just did it to be consistent with our other pkgs. Do you want me to revert?

Reply:
all fine